src/shared/lib/comparison/ComparisonStore.ts

```diff
@@ -235,12 +236,16 @@ export default abstract class ComparisonStore extends AnalysisStore
             );
         } else {
             // default
-            return mutationEventTypeSelectInit(
-                this.alterationEnrichmentProfiles.result?.mutationProfiles || []
-            );
+            return this.defaultMutationEnrichmentEventTypes;
         }
     }
 
+    @computed get defaultMutationEnrichmentEventTypes() {
```

Comment:
lets not call this this default since it is only meant distinguish from the table level.

Reply:
mutationEnrichmentEventTypes